components/frontend/src/report/Report.test.js

```diff
@@ -167,15 +167,3 @@ it("navigates to subject", async () => {
     expect(mockScroll).toHaveBeenCalledWith()
     expect(mockScrollBy).toHaveBeenCalledWith(0, 163)
 })
-
-it("does not navigate to a subject that is hidden", async () => {
-    history.push("?metrics_to_hide=no_issues")
-    const mockScroll = jest.fn()
-    window.HTMLElement.prototype.scrollIntoView = mockScroll
-    const mockScrollBy = jest.fn()
-    window.scrollBy = mockScrollBy
-    renderReport({ reportToRender: report })
-    fireEvent.click(screen.getAllByText(/Subject title/)[0])
-    expect(mockScroll).not.toHaveBeenCalled()
-    expect(mockScrollBy).not.toHaveBeenCalled()
```

Comment:
This test is now obsolete because empty subjects are now hidden from the dashboard.